pyaedt/hfss.py

```diff
@@ -710,62 +724,60 @@ def assign_coating(
-            if self.materials[mat]:
+        if material:
+            if self.materials[material]:
                 props["UseMaterial"] = True
-                props["Material"] = self.materials[mat].name
+                props["Material"] = self.materials[material].name
             else:
                 return False
         else:
             props["UseMaterial"] = False
-            props["Conductivity"] = str(cond)
-            props["Permeability"] = str(str(perm))
-        props["UseThickness"] = usethickness
-        if usethickness:
+            props["Conductivity"] = str(conductivity)
+            props["Permeability"] = str(str(permittivity))
+        props["UseThickness"] = use_thickness
+        if use_thickness:
             props["Thickness"] = thickness
-        if usehuray:
+        if use_huray:
             props["Radius"] = str(radius)
             props["Ratio"] = str(ratio)
             props["InfGroundPlane"] = False
         else:
             props["Roughness"] = roughness
-            props["InfGroundPlane"] = isinfgnd
-        props["IsTwoSided"] = istwoside
+            props["InfGroundPlane"] = is_infinite_ground
+        props["IsTwoSided"] = is_two_side
 
-        if istwoside:
-            props["IsShellElement"] = issheelElement
+        if is_two_side:
+            props["IsShellElement"] = is_shell_element
         else:
-            props["IsInternal"] = isInternal
-        return self._create_boundary("Coating_" + listobjname[1:], props, "Finite Conductivity")
-
-    # TODO: Extract name and type from **kwargs to pass them to create_setup() as setuptype and setupname
+            props["IsInternal"] = is_internal
+        if not name:
+            name = "Coating_" + listobjname[1:]
+        return self._create_boundary(name, props, "Finite Conductivity")
 
-    @pyaedt_function_handler()
-    def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
+    @pyaedt_function_handler(setupname="name", setuptype="setup_type")
+    def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         """Create an analysis setup for HFSS.
-        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
-        names correspond to the ``setuptype``
-        corresponding to the native AEDT API.  The list of
-        keywords here is not exhaustive.
+        Optional arguments are passed along with ``setup_type`` and ``setup_name``. Keyword
+        names correspond to the ``setup_type`` corresponding to the native AEDT API.
+        The list of keywords here is not exhaustive.
 
         .. note::
            This method overrides the ``Analysis.setup()`` method for the HFSS app.
 
         Parameters
         ----------
-        setuptype : str, optional
+        name : str, optional
+            Name of the setup. The default is ``"Setup1"``.
+        setup_type : str, optional
             Type of the setup. Based on the solution type, options are
```

Comment:
```suggestion
            Type of the setup, which is based on the solution type. Options are
```
Recommend moving the sentence about the default before the sentence listing the options.